### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by ra-rau**
Add documentation instead of Readme written in Sphinx.
Close issue#230

Signed-off-by: Ramona Rautenberg <rautenberg@osism.tech>

### CODE REVIEW

File: documentation/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    # 'canonical_url': 'https://docs.osism.de/',
+    'style_external_links': True,
+    'logo_only': True,
+    'prev_next_buttons_location': None
+}
+html_context = {
+    'display_github': True,
+    'github_user': 'osism',
+    'github_repo': 'openstack-image-manager',
+    'github_version': 'master',
```
Comment (berendt) [inline documentation/source/conf.py:36]: This should be main instand of master.

File: documentation/tox.ini [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+[testenv]
+basepython = python3
+
+[testenv:build]
+deps = -rrequirements.txt
+commands =
+    sphinx-build -b html source build/html
+
+[testenv:push]
```
Comment (berendt) [inline documentation/tox.ini:13]: The Push Environment Can be Removed.

File: documentation/source/index.rst [hunk 3]
```diff
@@ -0,0 +1,16 @@
+OpenStack Image Manager
```
Comment (berendt) [inline documentation/source/index.rst:1]: Add a ==== line First.

File: documentation/source/contribute.rst [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+==========
+Contribute
+==========
+
+To make changes in this repository, open a pull request. To prioritize the import
+of a new image send an email to `info@osism.tech` with reference to the created
+pull request.
+
+After creating a PR, please check the result of the Travis CI and correct any
```
Comment (berendt) [inline documentation/source/contribute.rst:11]: Today we use GitHub Actions.

File: documentation/source/configuration.rst [hunk 5]
```diff
@@ -0,0 +1,148 @@
+.. _configuration:
+
+=============
+Configuration
+=============
+
+After a change to the configuration, validate it with ``tox -e check``.
+
+Naming Convention
+-----------------
```
Comment (berendt) [inline documentation/source/configuration.rst:10]: Use ======= instead of ————-.

**Review by berendt — CHANGES_REQUESTED** [review]

File: documentation/source/conf.py (hunk 1 above)
Comment (ra-rau) [reply documentation/source/conf.py]: Done

File: documentation/tox.ini (hunk 2 above)
Comment (ra-rau) [reply documentation/tox.ini]: Done

File: documentation/source/index.rst (hunk 3 above)
Comment (ra-rau) [reply documentation/source/index.rst:2]: Done

File: documentation/source/contribute.rst (hunk 4 above)
Comment (ra-rau) [reply documentation/source/contribute.rst]: Done

File: documentation/source/configuration.rst (hunk 5 above)
Comment (ra-rau) [reply documentation/source/configuration.rst]: Done

**Pull request merged**